websites/H/HentaiLA/presence.ts

```diff
@@ -0,0 +1,72 @@
+const presence = new Presence({
+    clientId: "903690009633247252"
+  }),
+  browsingStamp = Math.floor(Date.now() / 1000);
+
+let video = {
+  duration: 0,
+  currentTime: 0,
+  paused: true
+};
+
+presence.on(
+  "iFrameData",
+  (data: { duration: number; currentTime: number; paused: boolean }) => {
+    video = data;
+  }
+);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "hentaila",
+    startTimestamp: browsingStamp
+  };
+  if (document.location.pathname === "/")
+    presenceData.details = "En la página de inicio";
+  else if (document.location.pathname.includes("/genero/")) {
+    presenceData.details = `Viendo género ${
+      document.querySelector(".section-title").textContent.split(":")[1]
+    }`;
+  } else if (document.location.pathname.includes("/ver/")) {
+    presenceData.details = "Viendo Hentai:";
+    presenceData.state = document.querySelector(".section-title").textContent;
+    [presenceData.startTimestamp, presenceData.endTimestamp] =
+      presence.getTimestamps(
+        Math.floor(video.currentTime),
+        Math.floor(video.duration)
+      );
```

Comment:
getMediaTimestamps?

Reply:
![imagen](https://user-images.githubusercontent.com/45316362/139599194-59388e11-22e9-4cf5-b80a-9d689a9607ad.png)
This function?

Reply:
getTimestampsFromMedia

Reply:
I think "getTimestamps" better to use and more easly 🤔.
It works perfect 😞💥

Reply:
No, use getTimestampsFromMedia()

Reply:
How it works

Reply:
https://docs.premid.app/en/dev/presence/class#gettimestampsfrommediahtmlmediaelement

Reply:
*Video is not videoelement.*Literally an object with duration, currently time and paused